### ISSUE TITLE
Audio fingerprinting instead

### ISSUE THREAD

**Issue by Protektor-Desura** [garrit-schroeder/tv-show-skipper#5]
It seems to me it would be much faster and just as reliable to have it scan the first X amount of time in every video and compare the AUDIO track and look for similar. This is going to work most of the time and it is how Plex and Emby do it. If you are looking at video frames that seems much slower and it still won't fix series like Simpsons where they change the intro multiple times in a season. Also it should add a intro chapter mark to every video to make it easy to skip to the next chapter and if it isn't perfect you can always speed ahead or rewind.

Also if you have a mix of 720p and 1080p files in a single season, then audio will be similar but a hash of video won't be.

**Comment by garrit-schroeder** [comment]
Sure, its a good idea. But a proof of concept of this idea is already available (In Emby & Plex). I wanted to create something with a different approach. That's all.

**Comment by rohitghali** [comment]
True. Audio hash is more logical. Less strain on the cpu. Better detection. Except a very very few shows like Seinfeld, audio is best.
